Issue4890 (#4907)
Codecademy/docs: content/markdown/concepts/links/links.md

## Accessibility: Link Titles

A title for a link can be optionally added. This will appear as a tooltip when the user hovers over the link. To add a title, enclose it in quotation marks `""` after the URL inside the parentheses.

Including link titles improves accessibility by providing additional context for screen readers and users navigating with the keyboard.

```md
[Link text](https://website-name.com "Link title")
```

For example:

```md
My favorite Craigslist category is [Missed Connections](https://newyork.craigslist.org/d/missed-connections/search/mis "The best place on the internet").
```

The output would look like:

My favorite Craigslist category is [Missed Connections](https://newyork.craigslist.org/d/missed-connections/search/mis "The best place on the internet").

## URLs and Email Addresses

To turn a URL or email address into a clickable link, enclose it in angle brackets `<URL or email>`.

- `<https://www.codecademy.com/resources/docs>`

> - `<foo42@hotmail.com>`

Reviewer: ```suggestion
- `<demoemail@hotmail.com>`
```

The rendered output looks like this:

- <https://www.codecademy.com/resources/docs>

> - <foo42@hotmail.com>

Reviewer: ```suggestion
- <demoemail@hotmail.com>
```

## Internal Linking

> To create links to navigate to other sections within the same document, or sections within other documents on the same website, use a link with the `#` symbol, followed by the section's title. Ensure the title is in lowercase, omit any punctuation and replace spaces with hyphens.

Reviewer 2: ```suggestion
To create links to navigate to other sections within the same document, or sections within other documents on the same website, use a relative path (when linking to sections in other documents) with the `#` symbol, followed by the section's title. Ensure the title is in lowercase, omit any punctuation and replace spaces with hyphens.
```

> To create links to navigate to other sections within the same document, or sections within other documents on the same website, use a relative path (when linking to sections in other documents) with the `#` symbol, followed by the section's title. Ensure the title is in lowercase, omit any punctuation and replace spaces with hyphens.

Reviewer: can you make this sentence short? or just divide it into multiple sentences.

Author: Hi, I'm new to open source so apologies in case I've missed anything!

I've split the sentence into two so it reads like this. Could also remove the bit in brackets, I think it's redundant but another reviewer added this earlier so I thought I should just leave it in for now.

```
To create links to navigate to other sections within the same document, or sections within other documents on the same website, use a relative path (when linking to sections in other documents) with the `#` symbol, followed by the section's title. Ensure the title is in lowercase, omit any punctuation and replace spaces with hyphens.
```

> For example, to link to the "Accessibility: Link Titles" section above:

Reviewer: ```suggestion
For example, to link to the `Accessibility: Link Titles` section above:
```

```md
[Accessibility: Link Titles](#accessibility-link-titles)
```

> 

Reviewer: ```suggestion

The output would look like:

```

The output would look like:

> [Accessibility: Link Titles](#accessibility-link-titles)

Reviewer: Put this under the pseudo block.

Author: What do you mean by pseudo block?

I guess maybe the code block, but the output example wouldn't format inside the code block.

Author: Got it, should be there now. Thank you :)

```

The output would look like:

My favorite Craigslist category is [Missed Connections](https://newyork.craigslist.org/d/missed-connections/search/mis "The best place on the internet").

## URLs and Email Addresses

To turn a URL or email address into a clickable link, enclose it in angle brackets `<URL or email>`.

- `<https://www.codecademy.com/resources/docs>`
- `<demoemail@hotmail.com>`

The rendered output looks like this:

- <https://www.codecademy.com/resources/docs>
- <demoemail@hotmail.com>

## Internal Linking

An internal link can be used to navigate to other sections within the same document, or sections within other documents on the same website. To add an internal link, use a relative path (when linking to sections in other documents) with the `#` symbol, followed by the section's title. Ensure the title is in lowercase, omit any punctuation and replace spaces with hyphens.

For example, to link to the `Accessibility: Link Titles` section above:

```md
[Accessibility: Link Titles](#accessibility-link-titles)
```

```pseudo

> The output would look like:
> 

Reviewer: ```suggestion
```
